### DISCUSSION TITLE
Secondary destructive buttons

### DISCUSSION THREAD

**Discussion by travisterwilligar** [twilio-labs/paste#636] [Ideas] [answered]
In some cases a destructive action may not be the primary CTA but would still be helpful to have alongside the primary. In those cases it would be really nice to have a destructive secondary button pattern.

**Comment by richbachman** [comment, marked as answer]
Hi @travisterwilligar. Thanks for the suggestion. We've been mulling this over as well, and we're all in agreement that a secondary destructive button should exist. I'm going to add a ticket to our backlog to get that created.
